x/strategicreserve/abci.go

```diff
@@ -14,3 +14,8 @@ func EndBlocker(ctx sdk.Context, k keeper.Keeper) {
 		panic(fmt.Sprintf("end block no %d failed : %s", ctx.BlockHeight(), err.Error()))
 	}
 }
+
+// BeginBlocker settles the active deposits of resolved order books
```

Comment:
can you explain what this begin blocker does?

Reply:
Feature Removed #153